packaging/deb/src/main/debian/debian-build.sh

```diff
@@ -68,22 +68,19 @@ cp $MOTECH_WAR $TMP_DIR
 cd $TMP_DIR
 
 # Create empty dirs if missing
-mkdir -p motech-base/var/cache/motech/motech-default/work/Catalina/localhost
-mkdir -p motech-base/var/cache/motech/motech-default/temp
-mkdir -p motech-base/var/cache/motech/motech-default/felix-cache
-mkdir -p motech-base/var/lib/motech/motech-default/webapps
-mkdir -p motech-base/var/lib/motech/motech-default/data/bundles
-mkdir -p motech-base/var/lib/motech/motech-default/data/rules
-mkdir -p motech-base/var/lib/motech/motech-default/data/config
-mkdir -p motech-base/var/log/motech/motech-default
-mkdir -p motech-base/usr/share/motech/motech-default
-
-# copy manage-tenant script
-cp $MOTECH_BASE/packaging/motech-manage-tenants motech-base/usr/share/motech/
+mkdir -p motech-base/var/cache/motech/motech/work/Catalina/localhost
```

Comment:
since we dropped tenants there is no point in using motech/motech, for example what's the point of having /var/log/motech/motech instead of /var/log/motech ?